readme.md

```diff
@@ -81,6 +81,21 @@ Default: `new nunjucks.Environment()`
 
 The custom Nunjucks [Environment object](https://mozilla.github.io/nunjucks/api.html#environment) which will be used to compile templates. If supplied, the rest of `options` will be ignored.
 
+#### filters
```

Comment:
`filters` should just be an option and not another argument. Just remove it from the object passed to Nunjucks.

Reply:
Alright. Should `options.env` also ignore `options.filters` then?

Reply:
Yes